Gruntfile.js

```diff
@@ -184,6 +184,27 @@ module.exports = function(grunt) {
     },
 
 
+    /* S3
+       Pushes image assets to an Amazon S3 bucket.
+       See: https://github.com/jpillora/grunt-aws
+    ------------------------------------------------- */
+
+    s3: {
+      options: {
+        accessKeyId: "<%= secret.s3.access_key_id %>",
+        secretAccessKey: "<%= secret.s3.secret_access_key %>",
+        bucket: "<%= config.s3.bucket %>",
+        region: "<%= config.s3.region %>",
+        overwrite: "<%= config.s3.overwrite %>"
+      },
+      build: {
+        cwd: "<%= config.s3.src %>",
+        src: "**",
+        exclusions: ["<%= config.s3.src %>/**/.DS_Store", "<%= config.s3.src %>/**/Thumbs.db"]
```

Comment:
Nice catch. I thought this worked but turns out that the plugin just ignores files starting in `.`